frappe/commands/utils.py

```diff
@@ -681,6 +654,10 @@ def auto_deploy(context, app, migrate=False, restart=False, remote='upstream'):
 	else:
 		print('No Updates')
 
+def print_in_an_help_deprecation():
```

Comment:
`print_in_app_help_deprecation`?

Reply:
@netchampfaris recommeded to have one single function to print out deprecation messages (link: https://github.com/frappe/frappe/pull/6801#discussion_r249880790)

This message is to inform users in case someone is trying to build the help database for their instance.

Reply:
yeah, the function name is being suggested.

Reply:
```suggestion
def print_in_app_help_deprecation():
```

Reply:
Fixed the typo.